eo-parser/src/main/antlr4/org/eolang/parser/Program.g4

```diff
@@ -284,11 +284,18 @@ BYTES:
     |  BYTE (MINUS BYTE)+;
 
 BOOL: 'TRUE' | 'FALSE';
-CHAR: '\'' ~'\'' '\'';
-STRING: '"' ('\\"' | ~'"')* '"';
+CHAR:  '\'' (~['\\\r\n] | ESCAPE_SEQUENCE) '\'';
+STRING: '"' (~["\\\r\n] | ESCAPE_SEQUENCE)* '"';
+
+fragment ESCAPE_SEQUENCE
+    : '\\' [btnfr"'\\]
+    | '\\' ([0-3]? [0-7])? [0-7]
+    | '\\' 'u'+ BYTE BYTE
```

Comment:
@andreoss are you sure this has to be `'u'+`? Maybe just `'u'`?

Reply:
@yegor256 I got it from here https://stackoverflow.com/questions/21522770/unicode-escape-syntax-in-java#21523055.